pytrip/utils/dddplot.py

```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/env python
+#
+#    Copyright (C) 2020 PyTRiP98 Developers.
+#
+#    This file is part of PyTRiP98.
+#
+#    PyTRiP98 is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU General Public License as published by
+#    the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    PyTRiP98 is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU General Public License for more details.
+#
+#    You should have received a copy of the GNU General Public License
+#    along with PyTRiP98.  If not, see <http://www.gnu.org/licenses/>.
+#
+"""
+Script for generating DVH or other volume histograms.
+"""
+import os
+import sys
+import logging
+import argparse
+import numpy as np
+import pytrip as pt
+
+logger = logging.getLogger(__name__)
+
+
+class DDD(object):
+    """This class handles DDD files."""
+    # TODO: this class could go into main pytrip/ as well.
```

Comment:
There is already a DDD class there:
https://github.com/pytrip/pytrip/blob/master/pytrip/ddd.py

also a new implementation ready here:
https://github.com/pytrip/pytrip/pull/339

I think that as for now we can keep it as it is, but merge these 3 concepts together

Reply:
moved to new issue Pushed to